composer.json

```diff
@@ -29,6 +29,7 @@
         "illuminate/support": "^5.0|^6.0|^7.0||^8.0"
     },
     "require-dev": {
+        "phpunit/phpunit": "^4.8.36 || ^5.7 || ^6.5",
```

Comment:
Adding the PHPUnit version semver to force  PHPUnit versions setup and fix following error message:

```
PHP Fatal error:  Declaration of Maherelgamil\Arabicdatetime\Tests\ArabicdatetimeTest::setUp() must be compatible with Orchestra\Testbench\TestCase::setUp(): void in /home/runner/work/arabicdatetime/arabicdatetime/tests/ArabicdatetimeTest.php on line 11
```